src/msgs.proto

```diff
@@ -1,12 +1,13 @@
-package gazebo_thruster.msgs;
+package gazebo_usv.msgs;
 
-message Thruster
+message DirectionedForce
 {
-    required string     name = 1;
-    optional double     effort = 2;
+    repeated double direction = 1;
+    optional double force = 2;
 }
 
-message Thrusters
+message ForceVectorLinkCmd
 {
-    repeated Thruster   thrusters = 1;
+    required string link_name = 1;
```

Comment:
You actually don't use the link name since there is a topic per link.

Reply:
I forgot to delete `ForceVectorLinkCmd`, DirectionedForce is the actual message I'm using